src/document-click.js

```diff
@@ -131,9 +131,17 @@ export function onDocumentElementClick_(e, viewport, history) {
   // to the custom protocol href.
   const isSafariIOS = platform.isIos() && platform.isSafari();
   const isEmbedded = win.parent && win.parent != win;
+  const isFTP = /^ftp:$/.test(tgtLoc.protocol);
+
+  // In case of FTP Links in embedded documents always open then in _blank.
+  if (isFTP && isEmbedded) {
+    win.open(target.href, '_blank');
+    e.preventDefault();
+  }
+
   const isNormalProtocol = /^(https?|mailto):$/.test(tgtLoc.protocol);
```

Comment:
I guess the variable name is not very helpful but no, FTP is a special case here and not a normal protocol.